tools/kicad/kicad_functions/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "kicad_functions"
```

Comment:
Any specific reason to make this a new crate compared to build into kicad_rs?

Reply:
Mostly just for easy expandability, one does not need to touch core application logic to extend its functionality (via adding these functions). IIRC the "external library" approach was also preferred when I asked about it.